Sources/Amplitude/Amplitude.m

```diff
@@ -513,6 +517,10 @@ - (void)logEvent:(NSString *)eventType withEventProperties:(NSDictionary *)event
     [self logEvent:eventType withEventProperties:eventProperties withGroups:nil];
 }
 
+- (void)logEvent:(NSString *)eventType withEventProperties:(nullable NSDictionary *)eventProperties extra: (nullable NSMutableDictionary *) extra {
+    [self logEvent:eventType withEventProperties:eventProperties withApiProperties:nil withUserProperties:nil withGroups:nil withGroupProperties:nil withTimestamp:nil outOfSession:NO extra:extra];
```

Comment:
Besides `outOfSession` it seems like all params start with "with". Could be good to stay consistent e.g. `withExtra` or `withMiddlewareExtra`

Reply:
Renamed the param to `withMiddlewareExtra`.